### PR TITLE
noise-repellent: init at unstable-29-12-2018

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/applications/audio/noise-repellent/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, meson, ninja, pkgconfig, fftwFloat, lv2 }:
+
+stdenv.mkDerivation rec {
+  pname = "noise-repellent";
+  version = "unstable-29-12-2018";
```
Comment (jonringer) [inline pkgs/applications/audio/noise-repellent/default.nix]: the date format is YYYY-MM-DD https://nixos.org/nixpkgs/manual/#sec-package-naming
```suggestion
  version = "unstable-29-12-2018";
```

Comment (magnetophon) [reply pkgs/applications/audio/noise-repellent/default.nix]: Thanks for the feedback.  Done.

**Review by jonringer — APPROVED** [review]
`nix-review` passes on NixOS
diff LGTM
no idea how to test

```
[3 built, 2 copied (1.3 MiB), 0.2 MiB DL]
https://github.com/NixOS/nixpkgs/pull/74204
1 package were built:
noise-repellent
```

**Comment by jonringer** [comment]
@GrahamcOfBorg build noise-repellent

**Comment by jonringer** [comment]
looks like arch doesn't like some of the compiler flags (architecture extensions), if arch is not a supported architecture, then I would change meta.platforms to reflect just the x86_64 variants

```
/nix/store/nalgbdpjmhizap3sp1q3lqinisifldl9-gcc-wrapper-8.3.0/bin/cc -Inrepel@sha -I. -I.. -I/nix/store/mwjc8kdg97nbl07bwsr96arkqmiz2v1w-fftw-single-3.3.8-dev/include -I/nix/store/6n1ib7m8axh6k75n6v9nkhbxp7s89ym6-lv2-1.16.0/include -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -std=c99 -fPIC -msse -msse2 -mfpmath=sse -ffast-math -fomit-frame-pointer -fno-finite-math-only -MD -MQ 'nrepel@sha/src_nrepel.c.o' -MF 'nrepel@sha/src_nrepel.c.o.d' -o 'nrepel@sha/src_nrepel.c.o' -c ../src/nrepel.c
gcc: error: unrecognized command line option '-msse'; did you mean '-fdse'?
gcc: error: unrecognized command line option '-msse2'
gcc: error: unrecognized command line option '-mfpmath=sse'
ninja: build stopped: subcommand failed.
builder for '/nix/store/7zy3w7g3mpk39zyr1ag7w8iwg364clg2-noise-repellent-unstable-2018-12-29.drv' failed with exit code 1
error: build of '/nix/store/7zy3w7g3mpk39zyr1ag7w8iwg364clg2-noise-repellent-unstable-2018-12-29.drv' failed
```

**Comment by magnetophon** [comment]
OK, platforms are updated.

**Pull request merged**